### PR TITLE
docs: add standard to managed mode migration doc

### LINKED ISSUE #917: Question: Migrating from Standard to Managed Mode
# Standard to Managed Mode migration steps

The managed mode docs show how to create a new podidentity setup in managed mode. However if one has an existing setup with "standard" mode and wants to migrate to "Managed"; would the below sequence be correct:

NOTE: `kubectl delete -f  https://github.com/Azure/aad-pod-identity/blob/master/deploy/infra/deployment-rbac.yaml`
This should not be done in a migration as this would delete the Managed Identity CRD and the "AzureIdentity" & AzureIdentityBindings" from every namespace

1. Run VMSS assignment to assign all identities to all scalesets.
 
2. Edit (I have NMI in a custom ns) and deploy only the managed mode deployment `kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/managed-mode-deployment.yaml`  

**OR**
              
`kubectl patch ds --patch ./managed-mode-depoyment.yaml`

3.  Delete the mic deployment, service accounts(`aad-pod-id-mic-service-account`), the `aad-pod-id-mic-role` clusterrole, clusterRoleBinding
4.  Apart from these 4 objects, is there anything else one needs to delete?

Would this be the right way to migrate without downtime (or with minimal)?

Would this require restarting the pods?
    I had to bounce my demo pod, but will try the entire operation again to confirm.

### LINKED ISSUE #1033: AKS clusters using system-assigned identity aren't supported?
Since seeing #973 mentioned in the 1.7.5 release notes, I've been trying to find some insight into why these AKS clusters aren't supported anymore, and what the change means. We currently use AKS clusters in this configuration, which as recently as last September @chewong [described as OK](https://github.com/Azure/aad-pod-identity/issues/804#issuecomment-697859092). 

We've been struggling with some pod-identity failures that seem consistent, but aren't reproducible by other pods in the same namespace assuming the same identity. It now sounds like we need to blow away our clusters and reproduce the problem before trying to pursue a resolution. That's going to take a while, so I'd love to know if our issues are plausibly related to 1.7.5's withdrawal of support.

The [role assignment documentation](https://azure.github.io/aad-pod-identity/docs/getting-started/role-assignment/) still describes system-assigned identity as a supported configuration, and the current version of the [hack/role-assignment.sh](https://github.com/Azure/aad-pod-identity/blob/28777f591c1be86006e8f2ef60fa73a53e0d1884/hack/role-assignment.sh) script doesn't raise an objection.

Thanks for your time.

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

**Reason for Change**:

**Requirements**

- [x] squashed commits
- [x] included documentation
- [x] added unit tests and e2e tests (if applicable). See [test standard](https://github.com/Azure/aad-pod-identity/blob/master/CONTRIBUTING.md#test-standard) for more details.
- [x] ran `make precommit`

**Issue Fixed**:

fixes #1033 
fixes #917 

**Please answer the following questions with yes/no**:

Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

- [ ] yes
- [x] no

**Notes for Reviewers**:

### CODE REVIEW

File: website/content/en/docs/_index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ menu:
 
 AAD Pod Identity enables Kubernetes applications to access cloud resources securely with [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) using User-assigned managed identity and Service Principal.
 
-> Note: System-assigned managed identity isn't supported with AAD Pod Identity.
+> Note: Configuring System-assigned managed identity with AAD Pod Identity to access cloud resources is not supported.
```
Comment (chewong) [inline website/content/en/docs/_index.md:11]:
```suggestion
> Note: Configuring system-assigned managed identity with AAD Pod Identity to access cloud resources is not supported.
```

File: website/content/en/docs/Configure/standard_to_managed_mode.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+> Available from 1.6.0 release
+
+## Introduction
+
+AAD Pod Identity supports 2 modes of operation:
+
+1. Standard Mode: In this mode, there is MIC and NMI components deployed to the cluster. MIC handles assigning/un-assigning the identity to the underlying VM/VMSS. NMI will intercept token request, validate if the pod has access to the identity it's requesting a token for and fetch the token on behalf of the application.
+2. Managed Mode: In this mode, there is only NMI. The identity needs to be manually assigned and managed by the user. Refer to [doc](../pod_identity_in_managed_mode) for more details on this mode.
```
Comment (chewong) [inline website/content/en/docs/Configure/standard_to_managed_mode.md:16]: dittos below

```suggestion
2. Managed Mode: In this mode, there is only NMI. The identity needs to be manually assigned and managed by the user. Refer to [this doc](../pod_identity_in_managed_mode) for more details on this mode.
```

**Review by chewong — COMMENTED** [review]
Few nits. Could we also add a `helm install/upgrade` command on how to install managed mode in https://azure.github.io/aad-pod-identity/docs/configure/pod_identity_in_managed_mode/?

**Comment by aramase** [comment]
>Few nits. Could we also add a helm install/upgrade command on how to install managed mode in https://azure.github.io/aad-pod-identity/docs/configure/pod_identity_in_managed_mode/?

Done!

**Review by chewong — APPROVED** [review]

**Pull request merged**